### PR TITLE
Add Istio 1.17.2

### PR DESCRIPTION
**Pull request by nicholasSUSE**
## Issue: 

https://github.com/rancher/rancher/issues/40959

## Solution
This PR adds Istio 1.17.2 version along with the latest Kiali server and Jaeger tracing versions.

Depends On:
- https://github.com/rancher/image-mirror/pull/372
- https://github.com/rancher/istio-installer/pull/64

Adds:
```
kiali v1.66
jaeger 1.43.0
istio:102.2.0+up1.17.2 (contains distro-less images)
```

More details on: https://github.com/rancher/image-mirror/pull/372/commits/20ad698324e33b5dd4d3bddd8c308d2c1b765019

### Checklist
- [ ] Install Repository 
- [ ] Install Istio
- [ ] Install Istio + Jaeger 
- [ ] Install Monitoring + Istio + Jaeger + Namespaces + P0 Scenario
- [ ] Uninstall 
- [ ] Reinstall 
- [ ] Update to the new version from 1.16.3 

### Results

### CODE REVIEW

**Comment by rohitsakala** [comment]
Installing and Uninstalling of Istio in AKS

Screenshot 2023-04-12 at 3 34 43 PM

Screenshot 2023-04-12 at 3 37 31 PM

File: packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 annotations:
   catalog.cattle.io/certified: rancher
   catalog.cattle.io/display-name: Istio
-  catalog.cattle.io/kube-version: '>= 1.22.0-0 < 1.26.0-0'
+  catalog.cattle.io/kube-version: '>= 1.23.0-0 < 1.27.0-0'
```
Comment (mitulshah-suse) [inline packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml:4]: 1.26 is not added for support yet. Ideally, the chart should be tested against a new version before adding it to the kube-version list.

**Comment by rohitsakala** [comment]
Tested Installation and Uninstallation of Istio on AKS kubernetes version 1.26.0 

Screenshot 2023-04-13 at 11 55 22 AM

Screenshot 2023-04-13 at 11 57 57 AM

File: packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml (hunk 1 above)
Comment (rohitsuse) [reply packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml:4]: I tested it on 1.26 kubernetes version and it works. @mitulshah-suse

Comment (mitulshah-suse) [reply packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml:4]: @prachidamle are we fine with adding support for k8s 1.26 before its introduced in Rancher?

**Comment by rohitsakala** [comment]
Upgrade from v1.16 to v1.17 works. 

Screenshot 2023-04-14 at 9 24 34 AM

**Comment by rohitsakala** [comment]
Upgrade from v1.15 to v1.17 works 

Screenshot 2023-04-14 at 9 30 57 AM

File: packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml (hunk 1 above)
Comment (prachidamle) [reply packages/rancher-istio/1.17/rancher-istio/charts/Chart.yaml:4]: yes this should be fine, as it just indicates this chart can work on k8s 1.26 cluster if the cluster gets upgraded to that version.

**Review by prachidamle — DISMISSED** [review]

**Review by rohitsakala — APPROVED** [review]

**Pull request merged**